client/Elements/Components/Topics/TopicsList.patternfly.tsx

```diff
@@ -194,7 +194,9 @@ export const TopicsList: React.FunctionComponent<ITopicList> = ({
           onPerPageSelect={onPerPageSelect}
           offset={0}
         />
+        </Card>
       )}
-    </>
+        </PageSection>
```

Comment:
Fix indentation for closing tags as well, along with the `<Card>` and `<Pagination>` in lines 187, 188.